### DISCUSSION TITLE
Windows RDP session inside teleport Desktop session

### DISCUSSION THREAD

**Discussion by chmi** [gravitational/teleport#37841] [Q&A] [answered]
I'm trying to start RDP connection to another Windows host from my teleport desktop session. But when i provide credentials (i'm using regular mstsc ) nothing is happening. RDP client becomes unresponsive and i got error that credential manager ui host is not responding.
Is it normal behavior ?

**Comment by zmb3** [comment]
Are you up to date? I recall an issue similar to this but it was fixed a while ago.

**Comment by chmi** [comment]
I'm on 15.0.0 now, have this problem since my first installations (14.0.0) , what version do you have ? (

**Reply by zmb3** [reply, marked as answer]
We tracked this one down and will release the fix in 15.1.0 today.
